<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookReferenceClass</class>
 <widget class="QDialog" name="BookReferenceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>177</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>177</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BookReference</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>306</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblName_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="txtBookName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblAuthor_2">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="txtAuthorName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblDescription_2">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="txtBookDescription"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnAdd">
        <property name="maximumSize">
         <size>
          <width>1677</width>
          <height>167</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnBookCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Edition</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="txtEdition"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnBookCancel</sender>
   <signal>clicked()</signal>
   <receiver>BookReferenceClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
